apis/infrastructure/v1beta1/byoclustertemplate_webhook.go

```diff
@@ -4,16 +4,11 @@
 package v1beta1
```

Comment:
possible conflict with PR #573 ?
I am referring to this comment - https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/pull/573#discussion_r885609665

Reply:
Yes I noticed it, but since we removed the `byocluster_webhook`, hence this change.

Reply:
What we can do is wait for #573 to merge maybe, before merging this one, wdyt @anusha94 ?

Reply:
I think we can merge this here and handle conflicts in that one. Basically, if the webhook is not doing anything, we should remove the file itself.